<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1058</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>611</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QPushButton" name="pencilButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/new/Icons/office-material.png</normaloff>:/new/Icons/office-material.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="eraserButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/new/Icons/eraser.png</normaloff>:/new/Icons/eraser.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="fillColorButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/new/Icons/artist.png</normaloff>:/new/Icons/artist.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="recToolButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/new/Icons/rectangle.png</normaloff>:/new/Icons/rectangle.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="ellipseButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/new/Icons/circle.png</normaloff>:/new/Icons/circle.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="lineButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/new/Icons/line.png</normaloff>:/new/Icons/line.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="penColor">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="clearButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font/>
     </property>
     <property name="text">
      <string>clear</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="penWidth">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>10</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="drawingbroad" name="drawingBoard" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>512</width>
      <height>512</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QScrollBar" name="verticalScrollBar">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>0</y>
       <width>17</width>
       <height>647</height>
      </rect>
     </property>
     <property name="maximum">
      <number>75</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QScrollBar" name="verticalScrollBar_2">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>0</y>
       <width>20</width>
       <height>611</height>
      </rect>
     </property>
     <property name="maximum">
      <number>75</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QScrollBar" name="horizontalScrollBar">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>590</y>
       <width>751</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="timelineWidget" native="true">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>20</y>
      <width>211</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QPushButton" name="addFrameButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>Add Frame</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteFrameButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Delete Frame</string>
     </property>
    </widget>
    <widget class="QPushButton" name="lastFrameButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>510</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/new/Icons/up-arrow.png</normaloff>:/new/Icons/up-arrow.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="showSpriteButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Show Sprite</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nextFrameButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>560</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/new/Icons/down-arrow.png</normaloff>:/new/Icons/down-arrow.png</iconset>
     </property>
    </widget>
    <widget class="framesarea" name="showFramesWidget" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>151</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="indexLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>53</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Current Frame:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1058</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_SSP"/>
    <addaction name="actionOpen_Image"/>
    <addaction name="actionSave"/>
    <addaction name="actionClose"/>
    <addaction name="actionhelp"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpen_Image">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionOpen_SSP">
   <property name="text">
    <string>Open SSP</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionhelp">
   <property name="text">
    <string>help</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>drawingbroad</class>
   <extends>QWidget</extends>
   <header location="global">drawingbroad.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>framesarea</class>
   <extends>QWidget</extends>
   <header>framesarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
